EurekaHelper/Windows/PluginWindow.cs

```diff
@@ -360,8 +360,15 @@ public void DrawTrackerTable()
 
             if (Connection.IsConnected())
             {
-                if (ImGui.BeginTable("TrackerTable", 5, ImGuiTableFlags.Resizable | ImGuiTableFlags.BordersInnerH | ImGuiTableFlags.BordersV | ImGuiTableFlags.NoBordersInBody | ImGuiTableFlags.ScrollY | ImGuiTableFlags.NoSavedSettings | ImGuiTableFlags.Sortable | ImGuiTableFlags.SortTristate))
+                var numColumns = 6;
+                if (ImGui.BeginTable("TrackerTable", numColumns, ImGuiTableFlags.Resizable | ImGuiTableFlags.BordersInnerH | ImGuiTableFlags.BordersV | ImGuiTableFlags.NoBordersInBody | ImGuiTableFlags.ScrollY | ImGuiTableFlags.NoSavedSettings | ImGuiTableFlags.Sortable | ImGuiTableFlags.SortTristate))
                 {
+                    var levelTableColumnFlags = ImGuiTableColumnFlags.WidthFixed;
```

Comment:
Suggest to add `ImGuiTableColumnsFlags.NoSort` flag to prevent the table column from being sortable as the sorting only caters for "Respawn In" column currently.

Reply:
Updated.